README.md

```diff
@@ -160,57 +161,13 @@ Successful response:
   }
 }
 ``` 
-🥰
-**Compliment**
-
-*'GET /api/v1/???'*
-
-Successful response:
-
-```
-{
-  "data": {
-      }
-}
-```  
-<br>
-
-🤬
-**Insult**
-
-*'GET /api/v1/???'*
-
-Successful response:
-
- ```
-{
-  "data": {
-      }
-}
-```  
-<br>
-
-📸
-**Picture**
-
-  *'GET /api/v1/???'*
-
-  Successful response:
-
- ```
-{
-  "data": {
-      }
-}
-``` 
-<br>
 
 🏁
 **Endgame**
-
-A request should include the number of wins and rounds 
-- The default number of wins = 0 and rounds = 5. 
-- It is recommended that wins are passed with the request and number of rounds adjust if different from the default 
+___
+- A request should include the number of wins and rounds, otherwise a default of wins = 0, rounds = 5 is provided
```

Comment:
Grammer adjustmentL "...otherwise the default value of wins = 0 and rounds = 5."